pom.xml

```diff
@@ -304,6 +304,26 @@
                     <version>${maven-coveralls-plugin.version}</version>
                 </plugin>
 
+
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>findbugs-maven-plugin</artifactId>
+                    <version>3.0.1</version>
+                    <configuration>
+                        <findbugsXmlOutput>true</findbugsXmlOutput>
+                        <xmlOutput>true</xmlOutput>
+                        <threshold>low</threshold>
+                        <excludeFilterFile>findbugs-exclude.xml</excludeFilterFile>
```

Comment:
Move the file to the misc/ directory:
`<excludeFilterFile>misc/findbugs-exclude.xml</excludeFilterFile>`